node/forest_libp2p/bitswap/README.md

```diff
@@ -0,0 +1,81 @@
+# forest_libp2p_bitswap
+
+Another libp2p bitswap([SPEC](https://github.com/ipfs/specs/blob/main/BITSWAP.md)) implementation in rust.
+
+## Features
+
+- Compatible with [`go-bitswap`](https://github.com/ipfs/go-bitswap)
+- Optional request manager
+- Prometheus metrics
+- Multiple async task API support, `async-std` and `tokio`(optional behind feature `tokio`)
+
+## Feature flags
+
+- `tokio`, disabled by default. Use task API(s) from `tokio` instead of `async-std` inside this crate.
+
+Note: since `async-std` task API(s) are compatible with `tokio` runtime, so it still works fine with `tokio` runtime when this feature is disabled. But it won't work with `async-std` runtime if this feature is enabled.
+
+## Usage
+
+Basic usage of `BitswapBehaviour`, checkout `tests/go_compat.rs` for details
+
+```rust
```

Comment:
maybe let's omit this, this can get outdated pretty fast since it's not compiled

Reply:
I thought the comments here would help, maybe remove this code section and move the comments to test code?

Reply:
Yes, sounds good. And point the users in the README to the relevant section in the code (or the file). It should be good enough and more maintainable.

Reply:
Fixed, also revised the readme a bit, making it clear that the builtin request manager is optimized for filecoin network